.travis.yml

```diff
@@ -1,8 +1,16 @@
-language: rust
-rust:
-  - 1.31.0
-cache: cargo 
-script:
-  - cd driver/
-  - cargo build --verbose --all
-  - cargo test --verbose --all
\ No newline at end of file
+matrix:
+  include:
+    - language: rust
+      rust:
+        - 1.31.0
+      cache: cargo 
+      script:
+        - cd driver/
+        - cargo build --verbose --all
+        - cargo test --verbose --all
+    - language: python
+      python: "3.7-dev"
```

Comment:
We are restricted to using python 3.6 because of our dependency on django-eth-events which only works in 3.6

Reply:
Why would it not work on something newer?